test/rest.test.js

```diff
@@ -1184,6 +1184,33 @@ describe('strong-remoting-rest', function() {
           .expect(500)
           .end(expectErrorResponseContaining({message: 'an error'}, done));
       });
+      
+      it('should return 500 for unhandled errors', function(done) {
```

Comment:
Please improve the test name and mention the important part - you are testing unhandled errors in `before` hooks.